scripts/cors_proxy.py

```
@@ -0,0 +1,85 @@
+            url,
+            data=body,
+            method=self.command,
+        )
+        # Forward relevant headers
+        for key in ("Content-Type", "Authorization", "Accept"):
+            val = self.headers.get(key)
+            if val:
+                req.add_header(key, val)
+
+        try:
+            with urllib.request.urlopen(req, timeout=120) as resp:
```

Ruff (0.14.14)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)